substratevm/mx.substratevm/mx_substratevm_benchmark.py

```diff
@@ -65,7 +65,11 @@ def list_jars(path):
                            '--allow-incomplete-classpath',
                            '--report-unsupported-elements-at-runtime',
                           ],
+    'finagle-chirper'   : [
+                            '-H:GenerateDebugInfo=0'
+                          ],
     'finagle-http'      : [
+                           '-H:GenerateDebugInfo=0',
```

Comment:
Why do we need this?

Reply:
In one of Scala classes inside this benchmark is declared field ε. For Scala, it's no problem because it's using UTF encoding, but the native image format doesn't support non-ASCII characters (https://github.com/oracle/graal/blob/master/substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/elf/dwarf/DwarfSectionImpl.java#L269-L269). First stacktrace is showing this exception thrown during image writing. I can leave a comment, to clarify.